### PR TITLE
Added PR #60 + formating

### PR DESCRIPTION
**Pull request by SkyzohKey**
Added File Transfers, File Resuming and Themes Support from the pull-request #60 + formating of the whole stuff (as requested by Zetok)

### CODE REVIEW

**Comment by SkyzohKeyx** [comment]
@GrayHatter @Impyy @tux3 @dvor @subliun can you please review this and vote ? :)

File: STS.md [hunk 1]
```diff
@@ -247,6 +295,14 @@ To validate this data a client would take the looked up user before the _tox and
 
 With Domain signing, the public key is also stored in a TXT record, using the format ```v=tox;pub={public key}```. It is important to note that due to potential issues wherein the public key may be the result of a poisoning attack, clients are encouraged to maintain a list of popular domains and keys. One such list is [here](http://wiki.tox.im/Domain_keys).
 
+## Customization
+####Themes support
```
Comment (GrayHatter) [inline STS.md]: This is single client, and shouldn't be in the STS

Comment (SkyzohKeyx) [inline STS.md]: What did u mean by "single client" ?

**Comment by GrayHatter** [comment]
I'm going to have to read though the whole STS first. Nothing mandatory should be in the STS. Only how to best interact with other clients.

**Comment by Impyy** [comment]
I don't understand why the File Transfer and File Resuming sections that you added should be in the STS. You're basically summarizing/explaining the documentation of the API and that's not what the STS is for.

I also don't agree with mandatory theme support, I personally don't see the need so I would rather see it as a recommended feature, not a mandatory feature. Also, how would a CLI client like toxic comply with this requirement?

**Comment by SkyzohKeyx** [comment]
@GrayHatter Ah! You're right, I'll remove the theming section.
@Impyy @zetok wanted a well-formated version of the #60 PR, so I did it. :p

**Comment by Impyy** [comment]
@SkyzohKeyx I see, I must have forgotten about that one.
Either way, I don't agree with this getting merged as it is now.

**Comment by zetok** [comment]
@SkyzohKeyx thanks for the PR. There's still missing https://github.com/Tox/Tox-STS/blob/f5d842541f213020ef09c95024d612a5299f2b2e/proposals.md 

You may want to look e.g. at #76 how commit message should be formatted, and perhaps amend commit in this PR to adhere?

**Pull request closed without merging**